relayer/crates/starknet-chain-components/src/types/cosmos/consensus_state.rs

```diff
@@ -10,7 +10,7 @@ use hermes_encoding_components::traits::transform::Transformer;
 #[derive(Debug, HasField)]
 pub struct CometConsensusState {
     pub timestamp: u64,
-    pub root: Vec<u8>,
+    pub root: [u32; 8],
```

Comment:
Shouldn't `CometUpdateHeader` also be updated accordingly?

Reply:
Done in ee085c68f267ab8c7fce9b099ef4da7084a9fd64